main.py

```diff
@@ -0,0 +1,16 @@
+from fastapi import FastAPI
+from sqlalchemy.orm import Session
```

Comment:
The import of `Session` from `sqlalchemy.orm` is not used in this file. Consider removing it to clean up the code and avoid confusion, especially since the application uses asynchronous sessions elsewhere.